cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+            3. Successful migration from vmware to OSP
+    """
+    pass
+
+
+@pytest.mark.tier(1)
+def test_osp_kill_the_v2v_process_in_the_middle_restart_evmserverd_should_resume_migration_pos():
+    """
+    title: OSP: kill the v2v process in the middle(restart evmserverd) - should
+    resume migration post restart
+
+    Polarion:
+        assignee: ytale
+        initialEstimate: 1/4h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: V2V
+        testSteps:
+            1. Create infrastructure mapping for vmware67 to OSP
+            2. Create migration plan with infra map
+            3. Start migration
+            4. restart evmserverd on appliance
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Successful migration from vmware to OSP
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_migration_plan_with_name_which_has_all_special_character():
+    """
+    title: OSP: Test migrating a VM using migration plan with name which has all
+    special characters
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware67 to OSP
+            2. Create migration plan with special chars in name
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration from vmware to OSP
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_flavors_can_be_selected_creating_migration_plan():
+    """
+    title: OSP: Test flavors can be selected creating migration plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware67 to OSP
+            2. Create migration plan amd choose flavors
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration from vmware to OSP with the selected flavor
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_creating_multiple_migration_plans_with_same_name():
+    """
+    title: OSP: Test duplicate names in creating migration plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create two migration plan with same name
+        expectedResults:
+            1.
+            2. Duplicate migration plan name should not be allowed.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vm_name_with_special_characters_can_be_imported():
```

Comment:
Since there is [BZ1685948](https://bugzilla.redhat.com/show_bug.cgi?id=1685948) there will be no use of this, remove this as well.